package.json

```diff
@@ -59,16 +59,16 @@
     "mocha": "3.0.2",
     "node-sass": "^3.13.0",
     "rimraf": "^2.5.2",
-    "rxjs": "5.0.0-rc.4",
+    "rxjs": "5.0.1",
     "ts-node": "1.2.2",
     "tslint": "^4.0.0",
     "typescript": "^2.1.4",
     "zone.js": "^0.7.2"
   },
   "peerDependencies": {
     "tslint": "^4.0.0",
-    "@angular/compiler": "<=2.3 >=2.2.0",
-    "@angular/core": "<=2.3 >=2.2.0"
+    "@angular/compiler": "<=2.4 >=2.2.0",
```

Comment:
I'm using some internal APIs of the compiler which often change between minor versions.